### DISCUSSION TITLE
阿里云同时显示资源盘和备份盘

### DISCUSSION THREAD

**Discussion by l21565** [AlistGo/alist#4968] [Ideas]
alist V3.25.1只能在资源盘和备份盘中二选一，要在alist配置中切换。能否像阿里云盘客户端那样同时显示资源盘和备份盘，很方便的切换。

**Comment by anwen-anyi** [comment]
应该不行，因为AList是一个列表，要怎么显示？
 - 而且备份盘是主要存储自己备份的内容的，不建议分享
 - 如果你只需要分享不要备份，哪你只用一个不就可以吗~用两个来回折腾也麻烦

如果你非要显示两个的话 还是建议你挂载两个吧 自己选一下类型
